safe/utilities/keyword_io.py

```diff
@@ -231,16 +232,17 @@ def copy_keywords(
                 keywords[key] = extra_keywords[key]
             write_iso19115_metadata(destination_file, keywords)
             # write_keywords_to_file(new_destination, keywords)
-        except Exception, e:
+        except Exception as e:
             message = self.tr(
                 'Failed to copy keywords file from : \n%s\nto\n%s: %s' % (
                     source_layer.source(), new_destination, str(e)))
             raise Exception(message)
         return
 
-# methods below here should be considered private
+    # methods below here should be considered private
```

Comment:
We are already doing that. I think this one is legacy and I changed the indent only